src/node.cc

```diff
@@ -1108,33 +1151,43 @@ InitializationResult InitializeOncePerProcess(
     OPENSSL_INIT_free(settings);
 
     if (ERR_peek_error() != 0) {
-      result.exit_code = ERR_GET_REASON(ERR_peek_error());
-      result.early_return = true;
-      fprintf(stderr, "OpenSSL configuration error:\n");
-      ERR_print_errors_fp(stderr);
+      // XXX: ERR_GET_REASON does not return something that is
+      // not useful as an exit code at all.
+      result->exit_code_ = ERR_GET_REASON(ERR_peek_error());
```

Comment:
Not sure what to do about this, I left this as is to avoid this becoming a breaking change. It’s definitely not right this way though :slightly_smiling_face:

Reply:
> Also nit: double negation in the comment

Fixed! :+1: